class-wc-facebookcommerce.php

```diff
@@ -183,6 +183,20 @@ public function add_admin_notices() {
 					'notice_class' => 'notice-info wc-facebook-migrate-notice',
 				] );
 			}
+
+			// if the connection is otherwise invalid, but there is an access token
+			if ( get_transient( 'wc_facebook_connection_invalid' ) && $this->get_connection_handler()->is_connected() ) {
+
+				$message = sprintf(
+					__( '%1$sHeads up!%2$s Your connection to Facebook is no longer valid. Please %3$sclick here%4$s to securely reconnect your account and continue syncing products.', 'facebook-for-woocommerce' ),
```

Comment:
Should we be adding translator comments?

Reply:
Good catch, added 27f90385076d771a97667df8129c96f9d5762e50